lib/config.js

```diff
@@ -1,54 +1,41 @@
-var mongoUrl = require('parse-mongo-url');
+var mongoUrl = require("parse-mongo-url");
```

Comment:
pls add this .prettierrc.json:
```json
{
  "printWidth": 100,
  "singleQuote": true
}
```

Reply:
sorry - create a file .prettierrc.json? did that and ran tests - it had no effect?

Reply:
doh! forgot the . in the filename - all good checking in with single quotes